frontend/components/draft/lesson/blocknote-editor-client.tsx

```
@@ -0,0 +1,64 @@
+import { BlockNoteView } from "@blocknote/mantine";
+import { blockNoteSchema } from "@/lib/blocknote/schema";
+import { useTheme } from "next-themes";
+import {
+  SuggestionMenuController,
+  getDefaultReactSlashMenuItems,
+} from "@blocknote/react";
+import { getCalloutSlashMenuItems } from "@/components/ui/blocknote/editor/slash-menu-config";
+import { getQuizSlashMenuItems } from "@/components/ui/blocknote/editor/slash-menu-config";
+
+interface BlockNoteEditorClientProps {
+  initialContent?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)